lib/receiver_link.js

```diff
@@ -192,7 +192,7 @@ ReceiverLink.prototype._messageReceived = function(transferFrame) {
   // @todo: Bump link credit based on strategy
 
   // respect settle mode in policy
-  if (this.policy.attach.rcvSettleMode === constants.receiverSettleMode.autoSettle) {
+  if (this.policy.settlement === constants.settlement.manual) {
```

Comment:
actually this should be
`if (this.policy.settlement === constants.settlement.manual) {`